sdk/storage/azure-storage-files-datalake/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 12.0.0-beta.6 (Unreleased)
```

Comment:
Date change here and `azure-storage-files-shares/CHANGELOG.md` is missing.

Reply:
@katmsft will do datalake service and file service and then update the date